src/main/java/com/qashqade/documentservice/service/WebMessageService.java

```diff
@@ -0,0 +1,41 @@
+package com.qashqade.documentservice.service;
+
+import com.qashqade.documentservice.model.Message;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+import org.springframework.web.reactive.function.client.WebClient;
+import reactor.core.publisher.Flux;
+import reactor.core.publisher.Mono;
+
+@Service
+public class WebMessageService {
+    @Autowired
+    private WebClient webClient;
+    @Autowired
+    private MongoMessageService mongoMessageService;
+
+    public Flux<Message> allMessages(int limit){
+        long currentTimeToGet = System.currentTimeMillis();
+        String queryParam = "";
+        if (limit>0) queryParam = "?limit="+limit;
+        Flux<Message> messageFlux = webClient.get()
+            .uri("/messages" + queryParam)
+            .retrieve()
+            .bodyToFlux(Message.class)
+            .doOnComplete(() -> System.out.println("documentservice " + limit + " Yes! " + (System.currentTimeMillis() - currentTimeToGet)/1000.0))
+            .doOnError(e -> System.out.println("documentservice " + limit + "No ;(. " + e.getMessage()));
```

Comment:
Incorrect log messages.

Reply:
Resolved and rewritten incorrect messages to more informative ones